pages/braille.js

```diff
@@ -45,10 +45,13 @@ export default function BraillePage() {
   const brailleNumbers = [1, 4, 2, 5, 3, 6]
   const [selected, setSelected] = useState(new Set())
   const [solutionText, setSolutionText] = useState(new String(''))
+  const [solutionBraille, setSolutionBraille] = useState(new String(''))
```

Comment:
Minimize state by storing solution in only one variable and then derive other forms (e.g. text or braille).